resources/assets/js/features/user/actions.js

```diff
@@ -51,4 +51,97 @@ export const doVerify = (email, token) => dispatch => {
             .then(response => dispatch(verifySuccess(response.data)))
             .catch(error => dispatch(verifyFailed(error.response.data)));
     }
+};
+
+
+export const loginSuccess = data => ({
+    type: actions.LOGIN_SUCCESS,
+    data
+});
+
+export const loginFormFailed = data => ({
+    type: actions.LOGIN_VERIFY_FAILED,
+    data
+});
+
+function processFailedLoginResponse(response) {
+    switch (response.status) {
```

Comment:
action shouldn't know about various status codes, probably it can be moved to component